symfony/framework-bundle/4.2/config/bootstrap.php

```diff
@@ -6,7 +6,8 @@
 
 // Load cached env vars if the .env.local.php file exists
 // Run "composer dump-env prod" to create it (requires symfony/flex >=1.2)
-if (is_array($env = @include dirname(__DIR__).'/.env.local.php')) {
+$phpEnvFile = dirname(__DIR__).'/.env.local.php');
+if (file_exists($phpEnvFile) && is_array($env = include($phpEnvFile)) {
```

Comment:
Hm, what if the file exists but could not be read due to strict permissions? Is it possible? Probably we still need `@include` here

Reply:
`file_exists` does i/o while `@include` does not. If supporting weird xdebug features goes against perf, we won't to it. This should be investiguated.

Reply:
@bocharsky-bw if the permissions are skewed than file_exists probably can't read it as well, since the permission won't allow it, I guess. @nicolas-grekas I think the case is not about some fancy xdebug feature. The questions comes down to if surpressing errors is in the interest of the user. If a user wishes to see `E_WARNINGS` that should be respected, shouldn't it?

Reply:
The author expressing failures should be ignored is what should have the highest precedence.